webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ContentViewDetailsCard.js

```diff
@@ -95,7 +95,103 @@ ContentViewEnvironmentDisplay.propTypes = {
   }).isRequired,
 };
 
-const HostContentViewDetails = ({
+export const CVEDetailsBareCard = ({
+  contentViewEnvironments, hostPermissions, permissions, dropdownItems,
+  isDropdownOpen, toggleHamburger,
+}) => {
+  const userPermissions = { ...hostPermissions, ...permissions };
+  const showKebab = hasRequiredPermissions(requiredPermissions, userPermissions);
+
+  return (
+    <Card ouiaId="content-view-details-card">
+      <CardHeader>
+        <Flex
+          alignItems={{ default: 'alignItemsCenter' }}
+          justifyContent={{ default: 'justifyContentSpaceBetween' }}
+          style={{ width: '100%' }}
+        >
+          <FlexItem>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <FlexItem>
+                <CardTitle>
+                  <FormattedMessage
+                    id="cv-card-title"
+                    defaultMessage="{count, plural, =0 {Content view environments} one {Content view environment} other {Content view environments}}"
+                    values={{
+                      count: contentViewEnvironments.length,
+                    }}
+                  />
+                </CardTitle>
+              </FlexItem>
+            </Flex>
+          </FlexItem>
+          {showKebab && dropdownItems && (
+            <FlexItem>
+              <Dropdown
+                toggle={<KebabToggle aria-label="change_content_view_hamburger" onToggle={toggleHamburger} />}
+                isOpen={isDropdownOpen}
+                isPlain
+                ouiaId="change-host-content-view-kebab"
+                position="right"
+                dropdownItems={dropdownItems}
+              />
+            </FlexItem>
+          )}
+        </Flex>
+      </CardHeader>
+      <CardBody>
+        <Flex direction={{ default: 'column' }}>
+          {contentViewEnvironments.map(env => (
+            <ContentViewEnvironmentDisplay
+              key={`${env.lifecycle_environment.name}-${env.content_view.name}`}
+              contentView={env.content_view}
+              lifecycleEnvironment={env.lifecycle_environment}
+            />
+          ))}
+        </Flex>
+      </CardBody>
+    </Card>
+  );
+};
+
+CVEDetailsBareCard.propTypes = {
+  contentViewEnvironments: PropTypes.arrayOf(PropTypes.shape({
+    content_view: PropTypes.shape({
+      name: PropTypes.string,
+      id: PropTypes.number,
+      composite: PropTypes.bool,
+    }),
+    lifecycle_environment: PropTypes.shape({
+      name: PropTypes.string,
+      id: PropTypes.number,
+    }),
+  })),
+  hostPermissions: PropTypes.shape({
+    edit_hosts: PropTypes.bool,
```

Comment:
needs to be tailored to work with AK

Reply:
In AK `CVEDetailsBareCard` is only for display, so we don't need to worry about permissions (yet).